doc/ReleaseNotes

```diff
@@ -31,6 +31,16 @@ TCPDirect distributed as source code
   packages are suitable for all supported OSs.
 
 
+New onload-devel package required
+---------------------------------
+
+  Onload now includes a package containing all of the header files used by
+  Onload. Installing this package is now required to build TCPDirect applications.
```

Comment:
I'm ok with this. As a subsequent editorial pass I'd be tempted to change to "containing headers required by ef_vi applications like TCP Direct" or similiar, as it may not always be "all of the header files used by Onload" - it's not actually ideal that we include headers that are really internal and that could change.

Reply:
> I'd be tempted to change to "containing headers required by ef_vi applications like TCP Direct" or similiar, as it may not always be "all of the header files used by Onload" - it's not actually ideal that we include headers that are really internal and that could change.

I strongly agree with this. I hadn't been paying enough attention to the packaging changes and only recently noticed that it was installing all the headers. If I'd noticed earlier I'd have grumbled about it at the time, but then deemed it too late by the point I noticed. This is definitely something that should be fixed before the next release though.

Reply:
Agreed. Thanks